doc/man/salt.7

```diff
@@ -288913,7 +288913,7 @@ data to return as pillar information. From there you can take a look at the
 varstack on how this file is evaluated.
 .INDENT 0.0
 .TP
-.B salt.pillar.varstack_pillar.ext_pillar(minion_id, pillar, conf)
+.B salt.pillar.varstack.ext_pillar(minion_id, pillar, conf)
 Parse varstack data and return the result
 .UNINDENT
```

Comment:
You do not need to make changes to this file.

We regenerate it with each release from all of the other doc strings.